app/modules/ads/tracking/page-tracker.js

```diff
@@ -33,6 +33,7 @@ export default {
       prop_name: name,
       prop_value: value,
       timestamp: (new Date()).getTime(),
+      tzOffset: (new Date().getTimezoneOffset()),
```

Comment:
Should we use camel case or underscore? 🙈

Reply:
Oh damn, I've already commited camelcase that to Crossbow repo 😱